internal/api/runme/runner/v1/runner.proto

```diff
@@ -177,6 +177,12 @@ message ExecuteRequest {
 
   // file extension associated with script
   string file_extension = 26;
+
+  // optional well known id for cell/block
+  string known_id = 27;
```

Comment:
It seems that it is not used. We can keep it for the completeness reason.

The most important for me is to provide more detailed description. What does "known" mean? What guarantees does it give? Maybe given an example how client would use this.

Finally, if you take a look at `config/v1alpha1/config.proto`, we have a way to validate field values in the proto definition. This is a bit tricky and not used anywhere else, so it might be a better idea for a separate PR.

Reply:
Will add an example. The reason why I added `id` (unused) was because I have near-term projects in mind where I need it. I'll be sure to make the description better.